### PR TITLE
request for dingding-notifications-plugin permission

### PR DESCRIPTION
**Pull request by ztbsuper**
request for permission for:https://github.com/jenkinsci/dingding-notifications-plugin

### CODE REVIEW

**Comment by daniel-beck** [comment]
Please follow the instructions in this repository's README.

File: permissions/component-dingding.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+name: "dingding notifier"
```
Comment (daniel-beck) [inline permissions/component-dingding.yml]: Wrong.

File: permissions/component-dingding.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+paths:
+- "org/jenkins-ci/plugins/dingding-notifications"
```
Comment (daniel-beck) [inline permissions/component-dingding.yml]: Wrong.

**Comment by ztbsuper** [comment]
Plz review again,  fixed the filename and the yaml content

**Comment by ztbsuper** [comment]
@daniel-beck I have no idea about the CI above, I saw the Artifact step failed, what should I do next?

**Comment by daniel-beck** [comment]
@ztbsuper Don't worry, @rtyler enabled PR builds for a repo that doesn't support them.

**Pull request merged**